### PR TITLE
chore: #product-20675 dependency update

### PR DESCRIPTION
**Pull request by pawel-zak**
#PRODUCT-20675
#PRODUCT-20220

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -1,153 +1,91 @@
...
 
-  let rampSdk: RampSdk = new RampSdk()
-    .on(RampEventTypes.WIDGET_CLOSE, (event) => {
-      widgetClose(event);
-      callbackText += JSON.stringify(event) + '\n\n';
-      console.log('CALLBACKTEXT: ' + callbackText);
-      setCallbackTextLabel(callbackText);
+    return new RampSdk({
+      url: 'https://app.demo.ramp.network',
+      hostAppName: 'Calcutta',
```
Comment (jakubsta) [inline example/src/App.tsx:17]: From where this name comes? Can it be like before?

**Review by jakubsta — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
+
```
[the remark this answers is not in the document]
Comment (pawel-zak) [reply CODE_OF_CONDUCT.md:2]: done

File: android/src/main/java/network/ramp/reactnativesdk/RampModel.kt [hunk 3]
```diff
@@ -94,19 +94,18 @@ object RampModel {
     purchaseMap.putString("receiverAddress", purchase.receiverAddress)
     purchaseMap.putString("cryptoAmount", purchase.cryptoAmount)
     purchaseMap.putString("fiatCurrency", purchase.fiatCurrency)
-    purchaseMap.putDouble("fiatValue", purchase.fiatValue.toDouble())
+    purchaseMap.putString("fiatValue", purchase.fiatValue.toString()) // Double?
```
[the remark this answers is not in the document]
Comment (pawel-zak) [reply android/src/main/java/network/ramp/reactnativesdk/RampModel.kt:97]: done

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
+# Contributing
+
+Contributions are always welcome, no matter how large or small!
```
[the remark this answers is not in the document]
Comment (pawel-zak) [reply CONTRIBUTING.md:3]: done

File: android/src/main/java/network/ramp/reactnativesdk/RampModel.kt (hunk 3 above)
[the remark this answers is not in the document]
Comment (pawel-zak) [reply android/src/main/java/network/ramp/reactnativesdk/RampModel.kt]: fixed

File: android/src/main/java/network/ramp/reactnativesdk/RampModel.kt [hunk 5]
```diff
@@ -94,19 +94,18 @@ object RampModel {
...
     purchaseMap.putString("paymentMethodType", purchase.paymentMethodType)
-    purchase.finalTxHash?.let{
-        purchaseMap.putString("finalTxHash", it)
-    }
+    // purchaseMap.putString("finalTxHash", purchase.finalTxHash) // ToDo Missing?
     purchaseMap.putString("createdAt", purchase.createdAt)
     purchaseMap.putString("updatedAt", purchase.updatedAt)
     purchaseMap.putString("status", purchase.status)
     purchaseMap.putString("escrowAddress", purchase.escrowAddress)
+    // purchaseMap.putString("escrowDetailsHash", purchase.escrowDetailsHash // ToDo Missing?
```
[the remark this answers is not in the document]
Comment (pawel-zak) [reply android/src/main/java/network/ramp/reactnativesdk/RampModel.kt]: fixed

File: example/src/App.tsx (hunk 1 above)
Comment (pawel-zak) [reply example/src/App.tsx]: fixed

File: android/src/main/java/network/ramp/reactnativesdk/RampModel.kt [hunk 6]
```diff
@@ -94,19 +94,18 @@ object RampModel {
...
+    purchaseMap.putString("fiatValue", purchase.fiatValue.toString()) // Double?
     purchaseMap.putDouble("assetExchangeRate", purchase.assetExchangeRate)
     purchaseMap.putDouble("baseRampFee", purchase.baseRampFee)
     purchaseMap.putDouble("networkFee", purchase.networkFee)
     purchaseMap.putDouble("appliedFee", purchase.appliedFee)
     purchaseMap.putString("paymentMethodType", purchase.paymentMethodType)
-    purchase.finalTxHash?.let{
-        purchaseMap.putString("finalTxHash", it)
-    }
+    // purchaseMap.putString("finalTxHash", purchase.finalTxHash) // ToDo Missing?
```
[the remark this answers is not in the document]
Comment (pawel-zak) [reply android/src/main/java/network/ramp/reactnativesdk/RampModel.kt]: fixed

File: android/src/main/java/network/ramp/reactnativesdk/RampSdkModule.kt [hunk 7]
```diff
@@ -42,7 +41,7 @@ class RampSdkModule(reactContext: ReactApplicationContext) :
 
 
     @ReactMethod
-    fun onOfframpCryptoSent(txHash: String, error: String) {
+    fun onOffRampCryptoSent(txHash: String, error: String) {
```
[the remark this answers is not in the document]
Comment (pawel-zak) [reply android/src/main/java/network/ramp/reactnativesdk/RampSdkModule.kt]: public

**Review by jakubsta — APPROVED** [review]

**Pull request merged**